CHANGELOG.md

```diff
@@ -1,2 +1,3 @@
-- Added emulator support for extensions defined by SDK.
-- Various emulator bug fixes.
+- Updated emulator UI to version 1.14.0, which adds support for SDK defined extensions.
+- Added emulator support for SDK defined extensions.
+- Fixed an issue where eventarc functions would not be emulated correctly after a reload.
```

Comment:
"Various emulator bug fixes" is perhaps less specific, but there were a lot off little things that I fixed with the functions emulator, not just correcting eventarc function removal.(Mostly to do with trigger handling,)

Reply:
Fair enough, i can switch it back or you can list out them more specifically if you think its worth it